<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    border-radius: 10px;
}
#formFrame{
border:2px solid rgba(255,255,255,0.5);
}

#hostnameLabel {
	color: white;
}

#formFrame { 
    background-color: rgba(80,80,80, 0.2);
    border-radius: 5px;
}

QComboBox,
QPushButton {
    color:black;
    border: 1px solid silver;
    background-color: rgba(200, 200, 200,0.5);
}




QComboBox::drop-down {
    border:none;
    width: 24px;
    text-align: right;
}

QComboBox::down-arrow {
    image: url(:/resources/dropdown.svg);
	color:red;
}
QLineEdit::focus{
border: 2px solid rgba(255, 236, 180 ,1);
	
}
QLineEdit{
color:white;
font-size:15px;
padding:5px;
border:1px solid white;
background-color: rgba(0,0,255,0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="formFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="kybrdcomboBox">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>60</height>
         </size>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="duplicatesEnabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="qt-lightdm-greeter.qrc">
           <normaloff>:/resources/keyboard.jpeg</normaloff>:/resources/keyboard.jpeg</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="NwpushButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt-lightdm-greeter.qrc">
          <normaloff>:/resources/rqt-2.png</normaloff>:/resources/rqt-2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+N</string>
        </property>
        <property name="autoRepeatDelay">
         <number>300</number>
        </property>
        <property name="autoRepeatInterval">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="leaveComboBox">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>60</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="qt-lightdm-greeter.qrc">
           <normaloff>:/resources/leaveIcon.svg</normaloff>:/resources/leaveIcon.svg</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qt-lightdm-greeter.qrc"/>
 </resources>
 <connections/>
</ui>
